xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -278,7 +278,7 @@ inline const char* GetTypeVersion(int type)
       return ADDON_INSTANCE_VERSION_VIDEOCODEC;
 #endif
   }
-  return "0.0.0";
+  return nullptr;
```

Comment:
**MAJOR BIG BUG**
This destroy all Kodi's with Version before this request.

There comes direct after binary addon load from repo (where created after this request) a crash on Kodi!

Reply:
In old addons this part is compiled in binary addon still with "0.0.0", no?

Reply:
Not after this request is in, Jenkins takes always the kodi master branch for his addons on Matrix.

Reply:
Things are clearified, PR #16641 fixes the unlikely situation,  old kodi + new addon at this code place:
https://github.com/xbmc/xbmc/blob/master/xbmc/addons/binary-addons/AddonDll.cpp#L477